Automattic/wp-calypso
# Podcasts: Update podcast categories to match Apple's update

#### Changes proposed in this Pull Request

In June, Apple updated and removed categories for Podcasts. This updates Calypso's embedded list of podcast categories to match that of Apple's.

#### Testing instructions

1. Starting at URL: https://wordpress.com/settings/podcasting/[site-address]
2. Go to section 'Podcast Topics'
3. Click on drop-downs there
4. Verify that podcast categories match [Apple's podcast categories](https://help.apple.com/itc/podcasts_connect/#/itc9267a2f12).

*

Fixes #35111
